src/gluonts/mx/block/scaler.py

```diff
@@ -174,6 +175,44 @@ def compute_scale(
 
         return F.maximum(scale, self.minimum_scale)
 
+class MinMax(Scaler):
+    """
+    The 'MinMax' scales the input data using a min-max approach along the specified axis. 
+    """
+
+    @validated()
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        
+    def compute_scale(self, F, data: Tensor, observed_indicator: Tensor) -> Tensor:
+
+        """
+        Parameters
+        ----------
+        F
+            A module that can either refer to the Symbol API or the NDArray
+            API in MXNet.
+
+        data
+            tensor containing the data to be scaled.
+
+        observed_indicator
```

Comment:
I think we need to take into account the observed indicator. NaN values will be replaced with dummy values (0 by default), which will impact the min/max values. Thus, the target values should be masked along the observed indicators to exclude them.

Reply:
Ok, I will replace the value of the scaled target with 0 where observed_indicator tells us to.

Reply:
Not sure we mean the same thing. Consider this case: 

```
target = [1, 2, 1, 0, 3]
observed_indicator = [1, 1, 1, 0, 1] 
```
then, the `min` value should be 1 (not 0) and the `max` value 3 (the target value with NaN will already be replaced with zero earlier in the transformation chain.